modules/installation-configure-proxy.adoc

```diff
@@ -39,7 +39,7 @@ endif::bare-metal[]
 +
 [NOTE]
 ====
-The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineCIDR`, `networking.clusterNetwork.cidr`, and `networking.serviceNetwork` fields from your installation configuration.
+The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineNetwork[0].cidr`, `networking.clusterNetwork[0].cidr`, and `networking.serviceNetwork` fields from your installation configuration.
```

Comment:
Should be `networking.machineNetwork[].cidr` and `networking.clusterNetwork[].cidr`, right?